deploy/clowdapp.yml

```
@@ -1438,6 +1438,69 @@ objects:
           requests:
             cpu: ${CPU_REQUEST_UNGROUPED_HOSTS_JOB}
             memory: ${MEMORY_REQUEST_UNGROUPED_HOSTS_JOB}
+    - name: delete-ungrouped-host-groups
+      restartPolicy: OnFailure
```

I would add a suspend flag with an environment variable set to False.

If we don't that, I think this job is going to be called as soon it reaches OCP

```suggestion
      restartPolicy: OnFailure
      suspend: ${{DELETE_UNGROUPED_SUSPEND}}
```

---

This is a regular job (no schedule). `suspend` is only supported for CronJobs afaik, so I don't think it's valid for this config. LMK if I'm mistaken though